### PR TITLE
Update Cachito API documentation

### PR DESCRIPTION
**Pull request by serg-cymbaluk**
CLOUDBLD-7666

Fix the documentation mismatches with the actual API.

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] OpenAPI schema is updated (if applicable)
- [ ] DB schema change has corresponding DB migration (if applicable)
- [ ] README updated (if worker configuration changed, or if applicable)

### CODE REVIEW

**Comment by brunoapimentel** [comment]
It's more of a nitpick, but I noticed that the lib that creates the API docs crops the titles at 51 chars, no matter the size of the screen:
image

The last 3 endpoints are all over that limit, which makes them harder to read.

I also noticed that the titles are supposed to be read from a key named  `summary`. Since we're only using `description`, the info there is copied to both the title and the description of each endpoint. 

It seems that the 51 chars limitation is only applied to the title (summary), not to the description, so we could use both keys in case we need more information while describing an endpoint.

**Comment by serg-cymbaluk** [comment]
@brunoapimentel 

Added summaries for endpoints.

File: cachito/web/static/api_v1.yaml [hunk 1]
```diff
@@ -558,6 +568,7 @@ paths:
                     example: The logs for the Cachito request 1 no longer exist
   "/requests/{id}/packages":
     get:
+      summary: List request packages and dependencies
```
Comment (brunoapimentel) [inline cachito/web/static/api_v1.yaml:571]: Maybe it would be less ambiguous if we kept it as `List packages and dependencies for a request`?

Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

File: cachito/web/static/api_v1.yaml [hunk 2]
```diff
@@ -693,7 +706,8 @@ paths:
 
   "/request-metrics/summary":
     get:
-      description: Return a summary about number and duration of requests
+      summary: Get summary of requests
```
[the remark this answers is not in the document]
Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

File: cachito/web/static/api_v1.yaml [hunk 3]
```diff
@@ -611,7 +622,8 @@ paths:
                     example: "Invalid state: packages file was not found."
   "/content-manifest":
     get:
-      description: Return the content manifest of the specified requests
+      summary: Get content manifest of multiple requests
```
[the remark this answers is not in the document]
Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

File: cachito/web/static/api_v1.yaml [hunk 4]
```diff
@@ -437,7 +444,8 @@ paths:
                     example: "Content manifests are only available for requests in the \"complete\" or \"stale\" states"
   "/requests/{id}/environment-variables":
     get:
-      description: Return information about the environment variables of a request
+      summary: List environment variables of request
```
[the remark this answers is not in the document]
Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

File: cachito/web/static/api_v1.yaml [hunk 5]
```diff
@@ -400,6 +406,7 @@ paths:
       - negotiateAuth: []
   "/requests/{id}/content-manifest":
     get:
+      summary: Get content manifest of request
```
[the remark this answers is not in the document]
Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

File: cachito/web/static/api_v1.yaml [hunk 6]
```diff
@@ -356,7 +361,8 @@ paths:
                     type: string
                     example: The requested resource was not found
     post:
-      description: Add configuration files to the request (requires special authorization)
+      summary: Add configuration files to request
```
[the remark this answers is not in the document]
Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

File: cachito/web/static/api_v1.yaml [hunk 7]
```diff
@@ -240,6 +242,7 @@ paths:
                     example: You are not authorized to create a request on behalf of another user
   "/requests/{id}":
     get:
+      summary: Get Cachito request
```
[the remark this answers is not in the document]
Comment (serg-cymbaluk) [reply cachito/web/static/api_v1.yaml]: fixed

**Review by brunoapimentel — APPROVED** [review]
LGTM

**Review by akhmanova — APPROVED** [review]
LGTM

**Pull request merged**